kawa/InputSourceManager.swift

```diff
@@ -88,19 +100,17 @@ class InputSourceManager {
             }
     }
     
-    static func tweak() {
-        selectPrevious()
-        Timer.scheduledTimer(timeInterval: 0.3, target: self, selector: #selector(selectPrevious), userInfo: nil, repeats: false)
+    static func englishSource() -> InputSource? {
+        return inputSources.first(where: { $0.id == "com.apple.keylayout.US" })
```

Comment:
Could it be made a `nonCJKSource` instead? Same way you check above for CJK, you could check here something like
```swift
inputSources.first($0.langs.notContains(where: { $0 == "ko" || $0 == "ja" || $0.hasPrefix("zh") }))
```

(not the real code, because I see that you need to get `langs somehow`), but I think you get the idea)

If it's slow to do such search every time, it could be somehow cached once found..